[2022-02-21 16:53:05,512] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-03T00:00:00+00:00 [queued]>
[2022-02-21 16:53:05,531] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-03T00:00:00+00:00 [queued]>
[2022-02-21 16:53:05,531] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 16:53:05,531] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 16:53:05,531] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 16:53:05,545] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-01-03 00:00:00+00:00
[2022-02-21 16:53:05,551] {standard_task_runner.py:52} INFO - Started process 62 to run task
[2022-02-21 16:53:05,554] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-01-03T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpff3s93tm', '--error-file', '/tmp/tmpyt1_p9o4']
[2022-02-21 16:53:05,554] {standard_task_runner.py:77} INFO - Job 5: Subtask get_path
[2022-02-21 16:53:05,607] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-01-03T00:00:00+00:00 [running]> on host 7c24e16e6467
[2022-02-21 16:53:05,664] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-01-03T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-03T00:00:00+00:00
[2022-02-21 16:53:05,665] {logging_mixin.py:109} INFO - /opt/***
[2022-02-21 16:53:05,665] {python.py:175} INFO - Done. Returned value was: None
[2022-02-21 16:53:05,678] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220103T000000, start_date=20220221T165305, end_date=20220221T165305
[2022-02-21 16:53:05,725] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 16:53:05,784] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-21 17:01:47,871] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-03T00:00:00+00:00 [queued]>
[2022-02-21 17:01:47,885] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-03T00:00:00+00:00 [queued]>
[2022-02-21 17:01:47,885] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 17:01:47,885] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 17:01:47,885] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 17:01:47,912] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-01-03 00:00:00+00:00
[2022-02-21 17:01:47,935] {standard_task_runner.py:52} INFO - Started process 62 to run task
[2022-02-21 17:01:47,938] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-01-03T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp_680ldle', '--error-file', '/tmp/tmpgw9ky8z2']
[2022-02-21 17:01:47,938] {standard_task_runner.py:77} INFO - Job 4: Subtask get_path
[2022-02-21 17:01:47,992] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-01-03T00:00:00+00:00 [running]> on host ce809d68944a
[2022-02-21 17:01:48,061] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-01-03T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-03T00:00:00+00:00
[2022-02-21 17:01:48,062] {python.py:175} INFO - Done. Returned value was: /nba_database
[2022-02-21 17:01:48,085] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220103T000000, start_date=20220221T170147, end_date=20220221T170148
[2022-02-21 17:01:48,111] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 17:01:48,161] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-21 19:10:32,797] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-03T00:00:00+00:00 [queued]>
[2022-02-21 19:10:32,808] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-03T00:00:00+00:00 [queued]>
[2022-02-21 19:10:32,808] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 19:10:32,808] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 19:10:32,808] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 19:10:32,821] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-01-03 00:00:00+00:00
[2022-02-21 19:10:32,826] {standard_task_runner.py:52} INFO - Started process 62 to run task
[2022-02-21 19:10:32,828] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-01-03T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpfbvxqo_p', '--error-file', '/tmp/tmprjp3sxow']
[2022-02-21 19:10:32,829] {standard_task_runner.py:77} INFO - Job 4: Subtask get_path
[2022-02-21 19:10:32,867] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-01-03T00:00:00+00:00 [running]> on host 1594888d6c38
[2022-02-21 19:10:32,947] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-01-03T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-03T00:00:00+00:00
[2022-02-21 19:10:32,948] {python.py:175} INFO - Done. Returned value was: ['/home/***/.local/bin', '/usr/local/lib/python37.zip', '/usr/local/lib/python3.7', '/usr/local/lib/python3.7/lib-dynload', '/home/***/.local/lib/python3.7/site-packages', '/usr/local/lib/python3.7/site-packages', '/opt/***/dags', '/opt/***/config', '/opt/***/plugins']
[2022-02-21 19:10:32,971] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220103T000000, start_date=20220221T191032, end_date=20220221T191032
[2022-02-21 19:10:33,002] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 19:10:33,028] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
